docs/CHANGELOG.asciidoc

```diff
@@ -58,6 +58,8 @@
 
 * Fix possible source of "Discarding sample = -nan(ind), weight = 1, variance scale = 1"
   log errors training classification and regression models. (See {ml-pull}2226[#2226].)
+* Fix possible source of "x = NaN, distribution = class boost::math::normal_distribution<..."
+  log errors training classification and regression models. (See {ml-pull}2249[#2249].)
```

Comment:
Since Easter is coming up and this still isn't merged it's likely to miss 8.2.0.  If it does slip to a subsequent release then please remember to move the release note.